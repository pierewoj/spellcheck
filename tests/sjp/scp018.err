1,25 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
3,17 Keter: eter geter heter kater keler ketem keten kuter peter seter teter
4,3 ppqtf: 
6,21 Przechowawcze: przechowawce
6,258 Przechowawcze: przechowawce
6,621 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
7,205 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
8,31 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
9,1 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
11,241 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
11,459 wewnątrzfundacyjną: 
13,43 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
13,202 Keter: eter geter heter kater keler ketem keten kuter peter seter teter
14,5 nzc: nbc nec nic noc nyc nzs
16,7 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
18,145 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
18,486 eksabajty: 
18,729 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
20,210 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
22,11 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
25,39 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
25,128 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
32,14 Władimirowicz: 
38,146 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
40,59 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
40,193 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
40,426 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
42,72 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
44,188 SCP: cp sap sbp sc schp sci sdp sep sęp sip sjp sp spp ssp syp tcp
46,257 Cięciewicz: 
