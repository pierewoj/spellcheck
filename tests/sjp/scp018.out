Identyfikator podmiotu: #SCP-PL-018

Klasa podmiotu: #Keter
15#ppqtf.jpg

Specjalne Czynności #Przechowawcze: Jako że osoba bądź anomalne źródło stojące za rozprzestrzenianiem podmiotu nie zostało poznane, jak również nieznane są metody jego przechowywania lub ograniczania wpływu w najmniejszym nawet zakresie, Specjalne Czynności #Przechowawcze nie mogły zostać sprecyzowane. Pracujący na terenie całej Polski agenci Fundacji czynnie przesłuchują dostawców Internetu oraz osoby trzecie wystawione na działanie podmiotu (aplikuje się im także preparaty amnezyjne klasy A, choć nie dowiedziono skuteczności ich stosowania). Podmiot stanowi olbrzymie zagrożenie szczególnie dla istnienia Fundacji #SCP, jako że w znacznym stopniu naraża organizację na dekonspirację poprzez udostępnianie jej zasobów wszelkim cywilom świadomym i nieświadomym manifestacji.
W związku z naturą podmiotu, a także niemożnością jego kontrolowania, odnalezienie obiektu (bądź ustalenie choćby przybliżonej jego lokalizacji) stało się priorytetowym zadaniem wszystkich filii Fundacji #SCP.
Będący świadkami manifestacji #SCP-PL-018 na lub poza terenem placówki członkowie personelu badawczego mają obowiązek bezzwłocznego zgłoszenia takiego zdarzenia jednemu z Administratorów O5, który zdecyduje o dalszym postępowaniu.
#SCP-PL-018 zaliczany jest do grupy obiektów konspiracyjnych.

Rozpoznanie: Aktywność podmiotu po raz pierwszy zaobserwowano w czerwcu 1995 roku w [UTAJNIONO] (Polska; stąd polski identyfikator podmiotu). Dr ███████, podczas rozmowy z synem, dowiedział się, iż ten odwiedził stronę internetową Fundacji #SCP. Zaskoczony tym wyznaniem (ponieważ dostęp do archiwów organizacji posiada jedynie wykwalifikowany personel), poprosił syna, by pokazał mu wspomnianą stronę. Okazało się, iż w rzeczy samej stanowiła ona oficjalną, #wewnątrzfundacyjną witrynę pozwalającą na dostęp do dokumentów ściśle tajnych bez jakiegokolwiek upoważnienia.
Doktor natychmiast zgłosił swoje odkrycie Administratorowi O5-██. Synowi doktora zaaplikowano preparat amnezyjny klasy A, jednak po dwóch dniach, kiedy nie zdawano sobie jeszcze sprawy z natury podmiotu, został zabity, zaś jego ciało zatopiono w kwasie po uprzednim umieszczeniu w komorze próżniowej. Dr ███████ otrzymał pochwałę i tydzień później desygnował podmiot.
Po upływie miesiąca Polska Filia Fundacji #SCP otrzymała oświadczenie o wystąpieniu manifestacji podmiotu na terenie Stanów Zjednoczonych, Japonii, Rosji, Francji i Chin. Obiektowi nadano wówczas klasę #Keter i podjęto próby jego neutralizacji, które do dziś nie przyniosły jednak zamierzonego skutku, wciąż ograniczając się do etapu poszukiwań.
28b8#nzc.jpg

Opis: #SCP-PL-018 to prawdopodobnie symulacja Internetu (zachowuje bowiem wszelkie jego cechy oraz zapewnia kompletny wachlarz funkcji) pochodząca z nieznanego źródła i stamtąd infekująca (podobnie do wirusa komputerowego) wybrane jednostki centralne. Infekcja nie wiąże się jednak bezpośrednio z oprogramowaniem zainstalowanym na komputerze, a raczej jego podzespołami. Obiekt objawia się w sposób świadomy (lub kontrolowany) wybranym cywilom, przypuszczalnie mając na celu przede wszystkim umożliwienie im uzyskania wglądu w dokumentację Fundacji i narażenie jej na dekonspirację.

Badania nad podmiotem pozwoliły ustalić, iż nie jest to jedyne odstępstwo od zachowania charakterystycznego dla klasycznej sieci internetowej — #SCP-PL-018 zdaje się bowiem komunikować z samym sobą za pomocą zaszyfrowanych, sprawiających wrażenie "zepsutych" plików graficznych (zwanych dalej pakietami pojemnościowymi). Dotychczas personel badawczy był w stanie stwierdzić, że pliki te, oprócz przekazywania wiadomości, służą podmiotowi do poszerzania swojej wewnętrznej przestrzeni o #eksabajty danych, dzięki czemu ulegać może ona stałemu, niekontrolowanemu rozrostowi, pozwalając tym samym na tworzenie jeszcze dokładniejszej symulacji. Nie wiadomo, w jaki sposób podmiot tworzy grafiki; nieznana jest także ich "droga" przez #SCP-PL-018.

Zgodnie z teorią doktora K. E. Maciejewskiego (badacz Polskiej Filii), pakiety pojemnościowe wytwarzane są za pomocą zewnętrznych serwerów wchodzących w skład podmiotu tylko pośrednio. Zdaniem doktora, źródło #SCP-PL-018 można sklasyfikować jako anomalia przejawiająca świadomość i stanowiąca większą, choć niecałkowitą część podmiotu; serwery zaś należeć mają do dostawców Internetu klasycznego, niebędących (bądź przeciwnie) świadomych zainfekowania przez obiekt.
Powyższą teorię traktować należy jednak z odpowiednim dystansem, bowiem opiera się jedynie o osobiste, niekoniecznie zgodne ze stanem faktycznym przypuszczenia dr. K. E. Maciejewskiego.
Istotnie, #SCP-PL-018 obecnie zaliczyć można do grona obiektów poznanych w minimalnym stopniu. Pomimo wspólnego wysiłku wszystkich filii, wciąż przyglądamy się jedynie czubkowi góry lodowej — dr K. E. Maciejewski.

Międzynarodowe badania nad podmiotem:
12/12/2010 Amerykańska Filia Fundacji #SCP opublikowała listę osób, które zdaniem jej analityków mogły paść ofiarą manifestacji #SCP-PL-018, a następnie wykorzystać zdobyte dzięki temu informacje dla osiągnięcia osobistych celów. Pośród nich znaleźli się między innymi:

    Bill Gates
    Usama ibn Ladin
    Timothy Berners-Lee
    Nikola Tesla
    █████ █████████ (ówczesny Administrator O5-05; został przesłuchany i zneutralizowany, gdyż na jaw wyszło, iż nie posiadał odpowiednich kwalifikacji pozwalających mu na pełnienie funkcji)
    Władimir #Władimirowicz Putin
    Stephen Hawking
    Michaił Bułhakow

W celu poznania treści pełnej wersji dokumentu, należy zwrócić się do Administratora O5-02.

Pracujący w Filii Chińskiej zespół naukowy, na czele z doktorem H. K. L. Hua, w roku 1996 sporządził obszerny dokument mający udowodnić tezę, iż #SCP-PL-018 istniał na długo przed wynalezieniem Internetu klasycznego, który przypuszczalnie został stworzony przez podmiot celem odwrócenia od niego uwagi. Publikacja dokumentu nastąpi w chwili, gdy przypuszczenia te znajdą swoje odbicie w rzeczywistości.

W roku 1997 Filia Rosyjska wyznaczyła do testów z użyciem #SCP-PL-018 grupę tysiąca członków personelu klasy D. Testerom oddano do dyspozycji komputery podłączone do wewnętrznej sieci Fundacji #SCP. Podmiot manifestował na jednej jednostce centralnej, co poskutkowało zyskaniem przez testera świadomości o własnym położeniu, a zakończyło się jego natychmiastową neutralizacją. Komputer zabezpieczono, jednak po dwóch godzinach #SCP-PL-018 przerwał manifestację. W wyniku tego wydarzenia wysnuto wniosek, iż podmiot nie tylko posiada świadomość, ale również potrafi uzyskać dostęp do każdej wybranej przez siebie jednostki centralnej.

Filia Japońska w latach 1998-2000 pracowała nad określeniem pojemności #SCP-PL-018, jednak z powodu braku wystarczającej wiedzy o obiekcie, prace te zawieszono. Obecnie pojemność obiektu określa się symbolem nieskończoności dzielonej przez dwa.

Polska Filia prowadzi z dostawcami Internetu na całym świecie negocjacje, których efektem ma być zastąpienie obecnych usług internetowych przez system od podstaw stworzony przez Fundację #SCP.

Na dwunastego października 2014 roku zaplanowano początek ofensywnej operacji mającej ograniczyć zakres oddziaływania podmiotu do minimum, skupiając się na eksterminacji wszystkich użytkowników wystawionych na jego choćby krótkotrwałe działanie — dr H. T. #Cięciewicz.
Operację anulowano z powodu zbyt wielkiej skali przedsięwzięcia i prawdopodobnie zbyt dużej liczby ofiar, poszukiwane jest rozwiązanie zastępcze — O5-05.
